tests/integration/lfs_getobject_test.go

```diff
@@ -89,6 +115,21 @@ func TestGetLFSSmall(t *testing.T) {
 	checkResponseTestContentEncoding(t, &content, resp, false)
 }
 
+func TestGetLFSSmallToken(t *testing.T) {
```

Comment:
`TestGetLFSSmallToken`

What is a "small token"?

Reply:
I think it's "Get LFS small (file with) Token"

Reply:
Hm, maybe it should be `with token` because there is a similar tests with username/password